financial-policy.md

```diff
@@ -0,0 +1,69 @@
+
+#### Project Closure:
+A project will be considered closed when all funds have been dispersed or when the Project Owner has informed the Treasurer that the funds are no longer required. The Treasurer will report any Project closures to the Board monthly and any funds returning to the general balance.
+
+#### Abandoned Project:
+A project will be considered abandoned when no funds have been dispersed after 90 days and no update has been provided to the Treasurer. The Treasurer will report any Abandoned Projects to the Board monthly and any funds returning to the general balance.
+
+### Recurring Operating Funds (AKA: Manager Budgets):
+#### Approval:
+When requested by an acting Manager, the Board can approve a recurring operating fund that a Manager can use at their discretion for their area of business. The Treasurer will confirm if funds are available in the general balance.
+
+#### Updates:
+The Treasurer will provide a monthly summary of all active Manager Budgets and their spending funds. The Treasurer will also check in with the acting Manager every ~ six months to ensure the funding is appropriate and will help guide a request to increase or decrease funds. Check-in can be skipped at the Treasurer's discretion.
+
+### Vending Machine Pool:
+In the case of the denhac-owned vending machines, a pool of funds will be set aside for stocking, maintenance, and general improvements. This pool will generate its funds through the machines' revenue until the approved cap is reached. Once reached, all collected funds will be added to the general fund. The Treasurer will report on the current funds in the pool monthly.
+
+The currently approved pool is $1,000.
+
+## Fund Disbursement:
+Project funds will be dispersed through reimbursement, a dedicated project card issued by Stripe, or direct-to-vendor payment made by the Treasurer.
+
+Manager Budgets will be dispersed through a dedicated card issued by Stripe under the Manager's name, reimbursement, or direct-to-vendor payment made by the Treasurer.
+
+A Project Owner can authorize a member to spend funds on behalf of their project. All reimbursement requests should be sent to treasurer@denhac.org. Reimbursements will be sent through PayPal; however, other reimbursement methods can be requested at the Treasurer's discretion.
+
+A Manager can authorize a member to spend funds under their budget. All reimbursement requests should be sent to treasurer@denhac.org. Reimbursements will be sent through PayPal; however, other reimbursement methods can be requested at the Treasurer's discretion.
+
+For reimbursement to be approved, the following will be required:
+- Budget Name
+- Manager CC'd (if not the submitter)
+- Area (If Applicable) (i.e., 3D Printer, Laser, Woodshop...)
+- Copy of Receipt
+- Total Amount to be reimbursed
+- Paypal address or preferred reimbursement method
+
+**A receipt will be required for all reimbursements with no exceptions.**
```

Comment:
I would say transactions, not just reimbursements here. ie even if Stripe card is used.